compiler/src/iree/compiler/Dialect/VM/Conversion/VMToEmitC/test/control_flow_ops.mlir

```diff
@@ -220,8 +220,7 @@ vm.module @my_module {
     // CHECK-SAME:     : () -> !emitc.opaque<"iree_string_view_t">
     // CHECK-NEXT: %[[MSGSIZE:.+]] = emitc.call "EMITC_STRUCT_MEMBER"(%[[MSG]]) {args = [0 : index, #emitc.opaque<"size">]}
     // CHECK-SAME:     : (!emitc.opaque<"iree_string_view_t">) -> !emitc.opaque<"iree_host_size_t">
-    // CHECK-NEXT: %[[MSGSIZEINT:.+]] = emitc.call "EMITC_CAST"(%[[MSGSIZE]]) {args = [0 : index, #emitc.opaque<"int">]}
-    // CHECK-SAME:     : (!emitc.opaque<"iree_host_size_t">) -> !emitc.opaque<"int">
+    // CHECK-NEXT: %[[MSGSIZEINT:.+]] = emitc.cast %[[MSGSIZE]] : !emitc.opaque<"iree_host_size_t"> to !emitc.opaque<"int">
```

Comment:
@henri-gruender You're change is mechanically correct. However, can you check if it should be
```suggestion
    // CHECK-NEXT: %[[MSGSIZEINT:.+]] = emitc.cast %[[MSGSIZE]] : !emitc.opaque<"iree_host_size_t"> to i32
````
This would of course require a change within the conversion as well and this can also be handled with a follow up PR. But please at least add it to our internal "we need to refactor this"-list so that we check.

Reply:
The type is now changed to i32.